packages/core/src/components/StructuredMetadataTable/StructuredMetadataTable.tsx

```diff
@@ -32,12 +32,13 @@ const listStyle = createStyles({
   },
 });
 
-const nestedListStyle = createStyles({
-  root: {
-    ...listStyle.root,
-    paddingLeft: '8px',
-  },
-});
+const nestedListStyle = (theme: Theme) =>
+  createStyles({
+    root: {
+      ...listStyle.root,
+      paddingLeft: theme.spacing(1),
```

Comment:
The indentation still looks a little bit off to me. I don't know if it might be a problem though :thinking: 

![Screenshot from 2020-04-27 13-09-20](https://user-images.githubusercontent.com/62359443/80405740-9f6ef400-8888-11ea-9413-f86249ee582e.png)